homeassistant/components/zwave_js/binary_sensor.py

```diff
@@ -50,11 +56,11 @@
 NOTIFICATION_GAS = "18"
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=True, kw_only=True)
 class NotificationZWaveJSEntityDescription(BinarySensorEntityDescription):
     """Represent a Z-Wave JS binary sensor entity description."""
 
-    off_state: str = "0"
+    not_states: set[str] = field(default_factory=lambda: {"0"})
     states: tuple[str, ...] | None = None
```

Comment:
Should we unify on either set or tuple here ?

Reply:
That will change in the next PR to a set.